subsys/nrf_security/src/drivers/cracen/Kconfig

```diff
@@ -22,6 +22,9 @@ config CRACEN_NEED_MULTIPART_WORKAROUNDS
 config CRACEN_NEED_IKG_INTERRUPT_WORKAROUND
 	def_bool SOC_NRF54LM20A || SOC_NRF54LV10A || SOC_NRF7120_ENGA
 
+config CRACEN_NEED_RNG_NO_ENTROPY_WORKAROUND
+	def_bool BOARD_NRF7120PDK_NRF7120_CPUAPP_EMU
```

Comment:
you can use newly introduced `SOC_NRF7120_PREENG` SoC variant, but I guess this passed without any WARs in FPGA?